test/Transforms/constant-fold.mlir

```diff
@@ -288,63 +288,63 @@ func @simple_diviu() -> (i32, i32, i32) {
 
 // -----
 
-// CHECK-LABEL: func @diviu_splat_tensor
-func @diviu_splat_tensor() -> (tensor<4xi32>, tensor<4xi32>, tensor<4xi32>) {
+// CHECK-LABEL: func @divi_unsigned_splat_tensor
+func @divi_unsigned_splat_tensor() -> (tensor<4xi32>, tensor<4xi32>, tensor<4xi32>) {
   %z = constant dense<0> : tensor<4xi32>
   // CHECK-DAG: [[C6:%.+]] = constant dense<6>
   %0 = constant dense<6> : tensor<4xi32>
   %1 = constant dense<2> : tensor<4xi32>
 
   // CHECK-DAG: [[C3:%.+]] = constant dense<3> : tensor<4xi32>
-  %2 = diviu %0, %1 : tensor<4xi32>
+  %2 = divi_unsigned %0, %1 : tensor<4xi32>
 
   %3 = constant dense<-2> : tensor<4xi32>
 
-  // Unsigned division interprets -2 as 2^32-2, so the result is 0.
+  // Unsigned divi_signedion interprets -2 as 2^32-2, so the result is 0.
```

Comment:
accidental changes?

Reply:
Yeah, I did the renaming with a script and only skimmed over the changes. I'll fix those.

Reply:
Fixed.